plugins/source/aws/codegen/recipes/base.go

```diff
@@ -277,9 +280,3 @@ func (r *Resource) generateMockTest(dir string) error {
 func (r Resource) StructName() string {
 	return reflect.TypeOf(r.Struct).Elem().Name()
 }
-
-// CloudQueryServiceName is used for accessing 'client.Services().{{.CloudqueryServiceName}}' in templates
-func (r Resource) CloudQueryServiceName() string {
-	csr := caser.New()
-	return csr.ToPascal(r.Service)
-}
```

Comment:
Maybe a small thing, but I think I'd rather keep this function so the logic is in one place, rather than spread across all the templates. As we saw recently, this does change from time to time. :)